wiki/en/Include-Backing-Up.md

```diff
@@ -1,15 +1,16 @@
 <!-- NOTE: This must apply to both Client and Server, and all operating systems -->
 
-Once installed and running, you may want to keep a copy of your settings. Having a backup is always a good idea, and settings files are not backwardly compatible between versions of Jamulus. So if you want to go back to the previous version, you will need to restore the settings you had. 
+Once installed and running, you may want to keep a copy of your settings. Having a backup is always a good idea, and settings files are not backwardly compatible between versions of Jamulus. So if you want to go back to the previous version, you will need to restore the settings you had.
 
-To find your settings file on Windows, look in the `%APPDATA%` folder. There will be one or more `.ini` files in a folder there named `Jamulus`.
+To find your settings file on Windows, type `%APPDATA%` into the search bar and look for a folder named `Jamulus`. There will be one or more `.ini` files in this folder. Now back up Jamulus' settings by copying these files to a different location.
```

Comment:
```suggestion
To find your settings file on Windows, type `%APPDATA%` into the search bar and look for a folder named `Jamulus`. There will be one or more `.ini` files in this folder. Now back up Jamulus' settings.
```

Reply:
Still not necessary to repeat that last sentence really (the para is about backing up), but hey.

Reply:
Yes. But it doesn’t hurt either.